pkgs/development/python-modules/pyod/default.nix

```diff
@@ -0,0 +1,98 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+, fetchpatch
+, joblib
+, matplotlib
+, numba
+, numpy
+, scikit-learn
+, scipy
+, six
+, pytestCheckHook
+, pandas
+, torch
+, statsmodels
+, xgboost
+}:
+
+buildPythonPackage rec {
+  pname = "pyod";
+  version = "1.1.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "yzhao062";
+    repo = "pyod";
+    rev = "v${version}";
+    hash = "sha256-nPFoN6Zd7kI2dVNGynidGy95n8zudpgLPGznohP7f4Q=";
+  };
+
+  patches = [
+    # Fix ImportError: cannot import name 'DistanceMetric' from 'sklearn.neighbors' error
+    (fetchpatch {
+      name = "fix-sklearn-distancemetric.patch";
+      url = "https://github.com/yzhao062/pyod/pull/513/commits/55ff0d45545da60237642d6d52d086ea4e33cd08.patch";
```

Comment:
Can we use this commit that is in the master branch? 
https://github.com/yzhao062/pyod/commit/f60813ff48527822c6a865f015a5e60f599116cb